### PR TITLE
password as crypto

### PR DESCRIPTION
**Pull request by Danilosimoes**

### CODE REVIEW

File: src/it/java/com/uvocab/api/controller/UserControllerTest.java [hunk 1]
```diff
@@ -11,13 +10,15 @@
...
 import org.springframework.http.ResponseEntity;
+import org.springframework.security.crypto.password.PasswordEncoder;
 import org.springframework.test.context.ActiveProfiles;
 import uvocab.protobuf.v1.User;
 
 @ActiveProfiles({"test"})
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 class UserControllerTest {
   @Autowired private TestRestTemplate restTemplate;
+  @Autowired private PasswordEncoder passwordEncoder;
```
Comment (robsonoduarte) [inline src/it/java/com/uvocab/api/controller/UserControllerTest.java:21]: we don't need this....

File: src/it/java/com/uvocab/api/controller/UserControllerTest.java [hunk 2]
```diff
@@ -44,13 +44,11 @@ void shouldPostTheUser() {
         User.newBuilder()
             .setLogin("danilo@uvocab.education")
             .setEmail("danilo@uvocab.education")
-            .setPassword("12345")
+            .setPassword("a")
```
Comment (robsonoduarte) [inline src/it/java/com/uvocab/api/controller/UserControllerTest.java:47]: is really necessary change this value ?

**Review by robsonoduarte — CHANGES_REQUESTED** [review]
some bit observation :police_car:

File: src/it/java/com/uvocab/api/controller/UserControllerTest.java (hunk 1 above)
Comment (Danilosimoes) [reply src/it/java/com/uvocab/api/controller/UserControllerTest.java:21]: I forgot to keep it the way it was.

**Review by robsonoduarte — APPROVED** [review]

**Pull request merged**